Headers/DebugServer2/Host/Windows/FileIO.h

```diff
@@ -25,13 +25,13 @@ namespace Windows {
 
 class FileIO {
 public:
-  static ErrorCode OpenFile(std::string const &path, uint32_t flags,
-                            uint32_t mode, int &fd);
-  static ErrorCode CloseFile(int fd);
+  ErrorCode OpenFile(std::string const &path, uint32_t flags, uint32_t mode,
```

Comment:
I like that you're making these non-`static`. Can you merge this work (of making it non-`static`) in the commit that introduces the class and make sure we have proper RTTI semantics on this?